src/jdmtool/featunlk.py

```diff
@@ -387,11 +388,20 @@ def display_content_of_dat_file(dat_file: pathlib.Path):
                 header_bytes = fd.read(0x200)
                 fd.seek(-0x102, os.SEEK_END)
                 footer_bytes = fd.read(0x102)
+    elif (feature in (Feature.CHARTVIEW,) ):
+        with open('Charts/chartview.hif', 'rb') as fd:
```

Comment:
The path needs to be relative to the original file - otherwise, it only works if you run `featunlk` from the directory with the featunlk.dat file. Something like `dst_file.parent / 'chartview.hif'` should work.

Reply:
Also, since it's the same offsets as for other features, you should just replace the file and keep the offset logic the same, i.e.:
```
    else:
        if feature == Feature.CHARTVIEW:
            dst_file = dst_file.parent / 'chartview.hif'
        with open(...) ...
```